.sonarcloud.properties

```diff
@@ -0,0 +1,5 @@
+# Define the root directory(s) for main sources and tests
+sonar.sources=MAP,API/src, API/src
```

Comment:
`API/src` is duplicated, should only have it once

Reply:
done!